app/event/send-enrich-request-blocked-event.js

```diff
@@ -1,8 +1,7 @@
 const raiseEvent = require('./raise-event')
 const getCorrelationId = require('../payment-request/get-correlation-id')
 
-const sendEnrichRequestBlockedEvent = async (paymentRequest) => {
-  const { paymentRequestId } = paymentRequest
+const sendEnrichRequestBlockedEvent = async (paymentRequestId, paymentRequest) => {
   if (paymentRequestId) {
     const correlationId = await getCorrelationId(paymentRequestId) ?? ''
```

Comment:
@steve-dickinson could you take a look at this file -- does the `id` of an event always need to be a `uuid`?

I have it as trying to read a `paymentRequest` `correlationId` but if doesn't have one, use its `paymentRequestId`, is this correct?

Reply:
I have created a tech debt ticket to answer and resolve this: https://eaflood.atlassian.net/browse/SFI-1954?atlOrigin=eyJpIjoiMDFhN2UwZjVlYjA5NDYyNmFkM2JjZDE4NTI4NTljYzkiLCJwIjoiaiJ9